utils.go

```diff
@@ -46,3 +48,18 @@ func promptForYes(message string) bool {
 	yn, _ := reader.ReadString('\n')
 	return strings.HasPrefix(yn, "Y") || strings.HasPrefix(yn, "y")
 }
+
+func generate_secret(length int) (string, error) {
+	chars := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789~!@#$%^&*()_+`-={}|[]\\:\"<>?,./"
```

Comment:
if you look at online generators, they offer the option to choose the set of characters (sometimes characters conflict with escaping chars). What if we add a config line to the .kiya config where they can override this.

Reply:
I was thinking options in invoking kiya, but indeed putting it in the configuration sounds cleaner and simpler.